config/crd/bases/toolchain.dev.openshift.com_tiertemplaterevisions.yaml

```diff
@@ -4,7 +4,6 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.12.0
-  creationTimestamp: null
```

Comment:
not sure why this has changed 🤔

Reply:
I guess that it's a result of a wrong conflicts solving in the @ranakan19's PR when she was updating the k8s version.

Reply:
see https://github.com/codeready-toolchain/host-operator/commit/c91f5c0cc879f2ee230877a04574bbd508c61f7b#diff-00ecc3755e3d97771f68b07c45dc5093ef0d59f0fd8a60e5df398c8f31f945d6L6

Reply:
ok, I see that the other `creationTimestamps` were removed in the PR , but this one no. Thanks!